src/components/List/List.tsx

```diff
@@ -142,9 +161,11 @@ class List extends UIComponent<Extendable<ListProps>, ListState> {
         const ref = React.createRef()
         this.itemRefs[idx] = ref
 
-        maybeSelectableItemProps.tabIndex = idx === selectedItemIndex ? 0 : -1
         maybeSelectableItemProps.ref = ref
         maybeSelectableItemProps.onFocus = () => this.focusHandler.syncFocusedItemIndex(idx)
+        maybeSelectableItemProps.onClick = () => this.trySetState({ selectedItemIndex: idx })
```

Comment:
shouldn't it be part of the accessibility behavior?

Reply:
```onClick``` handlers are part of component as it's functionality. I have added keydown handlers to accessibility behavior, to trigger this click handler when pressing ```Enter``` or ```Space```